react/features/transcribing/components/TranscribingExpandedLabel.native.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import { translate } from '../../base/i18n';
+import { ExpandedLabel, type Props as AbstractProps } from '../../base/label';
+
+type Props = AbstractProps & {
+    t: Function
+}
+
+/**
+ * A react {@code Component} that implements an expanded label as tooltip-like
+ * component to explain the meaning of the {@code VideoQualityLabel}.
+ */
+class VideoQualityExpandedLabel extends ExpandedLabel<Props> {
```

Comment:
This feels really weird. Can you rename this class to mention transcribing?

Reply:
It doesn't feel weird at all. This is a great example of non-weird copy paste mistake 😄